docs/spec.openapi.yaml

```diff
@@ -0,0 +1,627 @@
+              $ref: '#/components/schemas/authorization'
+      responses:
+        '200':
+          description: Account token is still valid
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/account'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+        '500':
+          description: Internal error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+  /account/password:
+    patch:
+      tags:
+        - Account
+      description: Updates the username password
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/authorization'
+                - type: object
+                  properties:
+                    currentPassword:
+                      type: string
+                    newPassword:
+                      type: string
+      responses:
+        '200':
+          description: Password updated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+        '500':
+          description: Internal error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+  /user/uuid/{username}:
+    get:
+      tags:
+        - Account
+      description: Obtain the uuid of an user by its username
+      parameters:
+        - in: path
+          name: username
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: User uuid
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/account'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+        '500':
+          description: Internal error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/statusResponse'
+  /files:
+    post:
+      tags:
+        - Files
+      description: Create a new file
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                fileName:
+                  type: string
+                location:
+                  type: string  
+                file:
+                  type: string 
```

Comment:
Do you plan to receive the file encoded in some way? I'm not saying the approach isn't valid, just keep in mind that the [CreateFileReq](https://github.com/hawks-atlanta/gateway-java/blob/dev/app/src/main/java/gateway/soap/request/CreateFileReq.java) in the SOAP service expects an array of bytes.